### PR TITLE
port to python 38

### PR DESCRIPTION
**Pull request by bissenbay**
Related-To: https://github.com/thoth-station/thoth-application/issues/545

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -17,3 +17,6 @@ pytest = "*"
 pytest-timeout = "*"
 pytest-cov = "*"
 coala-bears = "*"
+
+[requires]
+python_version = "3.8"
```
Comment (bissenbay) [inline Pipfile:22]: @fridex This repo didn't have `[requires]`, but I added, WDYT?

**Comment by bissenbay** [comment]
@fridex @harshad16 I have spent huge amount of time fixing `pre-commit` issues in this repo and really couldn't do much, could you please help with that?

File: Pipfile (hunk 1 above)
Comment (fridex) [reply Pipfile:22]: Nice catch. Yes, It is good to have it to make sure the right python version is running. :+1:

**Review by fridex — APPROVED** [review]
lgtm :100: :+1:

**Comment by fridex** [comment]
@bissenbay Please update also .thoth.yaml.

**Comment by bissenbay** [comment]
> @bissenbay Please update also .thoth.yaml.

sure thanx!

**Pull request merged**